packages/core/src/App/Containers/ResetEmailModal/reset-email-modal.jsx

```diff
@@ -70,15 +41,16 @@ const ResetEmailModal = ({
 
     const reset_initial_values = { email: '' };
 
-    if (is_send_email_modal_open) {
+    if (is_confirm_email_modal_open) {
         return (
-            <SentEmailModal
-                is_open={is_send_email_modal_open}
-                onClose={() => setIsSendEmailModalOpen(false)}
-                identifier_title={'Change_Email'}
-                onClickSendEmail={resendEmail}
-                has_live_chat={true}
-                is_modal_when_mobile={true}
+            <ConfirmEmailModal
+                changed_email={new_email}
+                is_open={is_confirm_email_modal_open}
+                onClose={() => setIsConfirmResetEmailModal(false)}
+                prev_email={email}
+                setIsEmailError={setHasError}
```

Comment:
rename this prop to `setHasError` all over the code as it needs to be more generic.

Reply:
Done